lib/src/ruby/ruby.cc

```diff
@@ -18,14 +18,14 @@ static const char load_puppet[] =
 "  $LOAD_PATH << Puppet[:libdir]\n"
 "end\n"
 "Facter.reset\n"
+"Facter.search_external([Puppet[:pluginfactdest]])\n"
```

Comment:
We find pluginsynced custom facts, but not custom facts for modules in codedir. This was a decision carried forward from 2.x. This commit doesn't change that, it just rearranges code.